### PR TITLE
as UV and Air Quality is added to the weather code, i had to make few changes in the unit test

### PR DESCRIPTION
**Pull request by SriLakshmiPolavarapu**
UV index and Air quality are the child components of weather so I had to add that too in the unit test for passing all the cases.

### CODE REVIEW

File: src/Components/weather.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from "react";
 import AirQuality from "./airQuality";
-import "../index.css";
+//import "../index.css";
```
Comment (glatimer) [inline src/Components/weather.jsx:3]: Is this commented out?  Does it need to be removed?

Comment (SriLakshmiPolavarapu) [reply src/Components/weather.jsx:3]: Oh actually if I have that in the code, it is not passing in unit test cases.

Comment (SriLakshmiPolavarapu) [reply src/Components/weather.jsx:3]: But there's no problem with styling if I remove calling the index.css

Comment (glatimer) [reply src/Components/weather.jsx:3]: okay!  If you want to remove it to clean it up that would be good

**Review by glatimer — APPROVED** [review]

**Pull request merged**